src/index.js

```diff
@@ -1,3 +1,50 @@
 document.addEventListener("DOMContentLoaded", () => {
-  // your code here
+  //grabs all the necessary DOM elements.
+
+  //set the taskForm to a variable.
+  const newTaskForm = document.querySelector('form')
+
+  //attach a submit event listener to taskForm.
+  newTaskForm.addEventListener('submit', (e) => {
+    e.preventDefault();
+    //create a variable called toDo holds the value of the targeted input.
+    const newToDo = e.target.newtaskdescription.value
```

Comment:
We can modify this to `e.target['new-task-description'].value` to preserve proper HTML formatting.